src/main/java/org/folio/orders/utils/PoLineCommonUtil.java

```diff
@@ -79,6 +79,24 @@ public static boolean isReceiptNotRequired(CompositePoLine.ReceiptStatus receipt
     return receiptStatus == CompositePoLine.ReceiptStatus.RECEIPT_NOT_REQUIRED;
   }
 
+  public static Physical getPhysical(PoLine poLine) {
+    return getPhysical(convertToCompositePoLine(poLine));
+  }
```

Comment:
Can we move after place where it is actually used or move most of get method located and make closer these similar methods
getPhysical(PoLine poLine) {
getPhysical(CompositePoLine compPOL) {
}
If yes, this is need for getEresource( method groun

Reply:
I like your strategy and I'm using it but not for static classes as most of it's methods are used from outside. I replaced them.